plugins/modules/get_certificate.py

```diff
@@ -294,6 +306,9 @@ def main():
         if proxy_host:
             module.fail_json(msg='To use proxy_host, you must run the get_certificate module with Python 2.7 or newer.',
                              exception=CREATE_DEFAULT_CONTEXT_IMP_ERR)
+        if ciphers:
```

Comment:
I would use the same condition as below:
```suggestion
        if ciphers is not None:
```